package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-aria-menubutton",
-  "version": "4.3.1",
+  "version": "4.3.2",
```

Comment:
It would mean a bump to 5.0. Let's actually leave out version changes from this PR, though: I prefer to do that as part of the release process.